src/full/Agda/Syntax/Translation/ReflectedToAbstract.hs

```diff
@@ -136,9 +136,14 @@ instance ToAbstract r => ToAbstract [Arg r] where
 -- instance ToAbstract r Expr => ToAbstract (Dom r, Name) (A.TypedBinding) where
 instance (ToAbstract r, AbsOfRef r ~ Expr) => ToAbstract (Dom r, Name) where
   type AbsOfRef (Dom r, Name) = A.TypedBinding
-  toAbstract (Dom{domInfo = i,unDom = x, domTactic = tac}, name) = do
+  toAbstract (Dom{domInfo = i, domIsFinite = isfin, unDom = x, domTactic = tac}, name) = do
     dom <- toAbstract x
-    return $ mkTBind noRange (singleton $ unnamedArg i $ mkBinder_ name) dom
```

Comment:
Anyone know why this discards the tactic attribute?

Reply:
Oh.. it's because it's coming from the _reflected_ syntax, and reflection can't make `pi`s with tactic annotations. I guess that makes sense: should we remove the binding for `tac`, or throw an `__IMPOSSIBLE_VERBOSE__` when it's set, or something? Strictly speaking it's impossible for `isFinite` to be `True`, here, too